docs/DOC207.md

```diff
@@ -21,12 +21,34 @@ The documentation contains a `<see langword="..."/>` element with an unrecognize
 
 ## Rule description
 
-*TODO*
+A violation of this rule occurs when documentation contains a `see langword` element but the reference is not recognized
+as a language keyword.
+
+```csharp
+/// <summary>
+/// This variable is <see langword="signed">.
+/// </summary>
+public int signed;
```

Comment:
`signed` is a keyword in C#, so this will not compile in the current form. The simplest fix is:

```suggestion
public int @signed;
```

Reply:
Hah... my C# is clearly rusty. That makes this a bad example. I'll go back to the drawing board!